src/dil_demo/web_form.clj

```diff
@@ -0,0 +1,111 @@
+;;; SPDX-FileCopyrightText: 2024 Jomco B.V.
+;;; SPDX-FileCopyrightText: 2024 Topsector Logistiek
+;;; SPDX-FileContributor: Joost Diepenmaat <joost@jomco.nl
+;;; SPDX-FileContributor: Remco van 't Veer <remco@jomco.nl>
+;;;
+;;; SPDX-License-Identifier: AGPL-3.0-or-later
+
+(ns dil-demo.web-form
+  (:require [ring.middleware.anti-forgery :refer [*anti-forgery-token*]])
+  (:import (java.util UUID)))
+
+(def ^:dynamic *object* nil)
+
+(defn anti-forgery-input []
+  [:input {:name "__anti-forgery-token", :value *anti-forgery-token*, :type "hidden"}])
+
+(defmacro form [object attrs & body]
+  `(binding [*object* ~object]
+     [:form ~attrs
+      (anti-forgery-input)
+      ~@body]))
+
+(defn- ks->name [ks]
+  (if (vector? ks)
+    (loop [result     (name (first ks))
```

Comment:
Kan dit simpler? lijkt er op dat dit niet werkt als ks 1 element heeft.

```
(reduce (fn [result k]
                (str (name result) "[" (name k) "]"))
              ks)
```

of zoiets?